deployments/warp_routes/CBBTC/base-zeronetwork-config.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
```

Comment:
Recommend adding a logoURI field to all token entries if these will be used in a UI